microsoft-edge/devtools-guide-chromium/whats-new/2021/05/devtools.md

```diff
@@ -0,0 +1,228 @@
+
+Perform the following steps to open the getting started documentation page. These steps assume you are in Visual Studio Code, with no folder opened.
+
+1.  In the **Activity Bar** on the left, select the **Microsoft Edge Tools** button.  In the **Side Bar**, the **Microsoft Edge Tools: Targets** panel appears.
+
+    :::image type="complex" source="../../media/2021/05/edge-tools-extension-in-context-docs-initial-side-bar.msft.png" alt-text="The Microsoft Edge Tools: Targets panel in the Side Bar of Visual Studio Code, showing the Launch Instance button" lightbox="../../media/2021/05/edge-tools-extension-in-context-docs-initial-side-bar.msft.png":::
+       The **Microsoft Edge Tools: Targets** panel in the **Side Bar** of Visual Studio Code, showing the **Launch Instance** button
+    :::image-end:::  
+
+1.  In the **Side Bar**, select the **Launch Instance** button.  Microsoft Edge opens, showing a documentation page, which provides guidance and links for more information.  Also, in Visual Studio Code, DevTools opens.
+
+    :::image type="complex" source="../../media/2021/05/edge-tools-extension-in-context-docs-doc-page.msft.png" alt-text="The new documentation page that appears in the browser, about how to use Microsoft Edge Developer Tools for Visual Studio Code" lightbox="../../media/2021/05/edge-tools-extension-in-context-docs-doc-page.msft.png":::
+       The new documentation page that appears in the browser, about how to use Microsoft Edge Developer Tools for Visual Studio Code
+    :::image-end:::  
+
+1.  In the address bar, navigate to the project that you want to debug using Microsoft Edge Developer Tools for Visual Studio Code.
+
+
+#### The Side Bar now provides a UI instead of a blank panel, to help launch your project for debugging
+
+The Side Bar now provides UI instead of a blank panel, to help you launch your project when you want to debug it. 
+
+Perform the following steps to launch your project. These steps assume you are in Visual Studio Code, with no folder opened.
+
+1.  In the **Activity Bar** on the left, select the **Microsoft Edge Tools** button.  In the **Side Bar**, the **Microsoft Edge Tools: Targets** panel appears, showing a single button (**Launch Instance**).
+
+    :::image type="complex" source="../../media/2021/05/edge-tools-extension-in-context-docs-initial-side-bar.msft.png" alt-text="The Microsoft Edge Tools: Targets panel in the Side Bar of Visual Studio Code, showing the Launch Instance button" lightbox="../../media/2021/05/edge-tools-extension-in-context-docs-initial-side-bar.msft.png":::
+       The **Microsoft Edge Tools: Targets** panel in the **Side Bar** of Visual Studio Code, showing the **Launch Instance** button
+    :::image-end:::  
+
+1.  In the **Microsoft Edge Tools: Targets** panel, select the **open a folder** link, and open your project folder.  The **Microsoft Edge Tools: Targets** panel now shows two buttons: **Launch Instance** and **Generate launch.json**.
```

Comment:
I think this can just be the start of the steps in this section. I don't think we need the previous step.

Reply:
Keeping previous step, because the UI differs based on whether a folder is opened.